### ISSUE TITLE
Data viewer fails to display data frames in list columns above column 50

### ISSUE THREAD

**Issue by BastiHz** [rstudio/rstudio#5851]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.5019
    OS Version      : Windows 10 x64 (build 17763)
    R Version       : 3.6.1

### Steps to reproduce the problem

1. Create a data frame with 51 columns. The last column should be a list containing a small data frame:
```
foo <- as.data.frame(matrix(NA, ncol = 50, nrow = 1))
foo$list_col <- list(data.frame(a = 1:2, b = 3:4))
```
2. Run `View(foo)` to open the data frame in the data viewer.
3. Scroll to the right and click on the little arrows to see column 51.
4. Click on the little data frame in column 51.

### Describe the problem in detail
A new tab opens but it just shows a single `NA`. The console prints this: `View(foo[[50]][[1]])`

### Describe the behavior you expected
Instead of the NA I wanted to see the small data frame. Same as when running `View(foo[[51]][[1]])`.
I only encountered this problem when the list was column 51 or above. Compare with this:
```
bar <- foo[, c(51, 1:50)]
View(bar)
```
Here the list is the first column and clicking on the small data frame inside it works correctly.

**Comment by ronblum** [comment]
@BastiHz Thank you for reporting this issue! We'll review it as we continue our ongoing development of RStudio.

Reproducible with RStudio Desktop on MacOS 10.15.2 Catalina.

**Comment by dfalty** [comment]
Verified fixed 1.3.867
